src/components/tables/PoolsTable/TokenPills/HiddenTokensPills.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="relative flex mr-2 my-1">
+    <div class="pill" :style="{ zIndex: tokens.length }">
+      {{ $t('tokenPills.hiddenTokens', [tokens.length]) }}
+    </div>
+    <div
+      v-if="hasBalance"
+      class="balance-indicator"
+      :style="{ zIndex: tokens.length }"
+    />
+    <div
+      v-for="n in 2"
+      :key="`hidden-pill-${n}`"
+      class="pill hidden-pill"
+      :style="{
+        transform: `translateX(${n * 8}px)`,
+        zIndex: tokens.length - n
+      }"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue';
+
+import { PoolToken } from '@/services/balancer/subgraph/types';
+
+export default defineComponent({
+  name: 'HiddenTokensPills',
+
+  props: {
+    tokens: {
+      type: Array as PropType<PoolToken[]>,
+      required: true
+    },
+    hasBalance: {
+      type: Boolean,
+      default: false
+    }
+  }
+});
+</script>
+<style scoped>
+.pill {
+  @apply px-2 py-1;
+  @apply rounded-lg;
+  @apply bg-gray-50 dark:bg-gray-600 text-gray-500 dark:text-gray-400;
+  @apply text-sm;
+  box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.07);
```

Comment:
I actually started to like seperating the `@apply` props into multi-line for better context.

Reply:
does scoped make the styles not be available globally?

Reply:
@evgenyboxer I like doing this too.

@1saf yeah `scoped` prevents any classes specified in this component being global by prepending (or appending I'm not sure which) a unique key.